atomic_reactor/koji_util.py

```diff
@@ -85,6 +85,7 @@ def __init__(self, session, task_id, poll_interval=5):
         self.session = session
         self.task_id = task_id
         self.poll_interval = poll_interval
+        self.state = 'CANCELED'
```

Comment:
Because then the `.failed()` method would return False. The point of that method is to know whether we should proceed (really it should probably be named .succeeded()...).

https://github.com/projectatomic/atomic-reactor/blob/master/atomic_reactor/plugins/pre_add_filesystem.py#L292